### ISSUE TITLE
USER STORY: Model Optimization

### ISSUE THREAD

**Issue by yuyizhong** [yuyizhong/ML-Mildew-Detector-II#5]
As a **developer**, I can **train and fine-tune the machine learning model based on the optimal hyperparameters found** so that to **achieve an agreed performance criteria a 97% accuracy**.

**Acceptance Criteria**

- Nice Model learning curve both for training and validation data
- Minimum 97% accuracy achieved
